config/api.yml

```diff
@@ -42,6 +42,20 @@
     - :subcollection
     :verbs: *g
     :klass: Account
+  :ansible_repositories:
+    :description: Ansible Repositories
```

Comment:
I'm a little confused for a few reasons...
- Does it make sense for this endpoint Ansible specific?
- Do we usually look at only one subclass rather than the base class?
- The class doesn't match the endpoint / description